### ISSUE TITLE
broken since Mattermost v5.12

### ISSUE THREAD

**Issue by mthld** [wbernest/mattermost-plugin-rssfeed#14]
hi,

since 5.12 release the plugin seems broken:
```
2019-06-24T13:46:28.050+0200	error	plugin/health_check.go:95	Health check failed for plugin rssfeed, error: Plugin process not found, or not responding
2019-06-24T13:46:28.051+0200	error	mlog/log.go:172	RPC call OnDeactivate to plugin failed.	{"plugin_id": "rssfeed", "error": "connection is shut down"}
2019-06-24T13:46:28.073+0200	error	mlog/sugar.go:23	Failed to create RSSFeed Plugin. Attempting to find existing one.	{"plugin_id": "rssfeed", "err": "SqlUserStore.Save: An account with that email already exists., user_id=d8umjyzba38k9n9mh4kh7z6a9a, pq: duplicate key value violates unique constraint \"users_email_key\""}
```

maybe it's related to how #13 was handled?

**Comment by bbodenmiller** [comment]
Tried 0.0.6 and 0.1.0 here... both broken.

**Comment by mthld** [comment]
Fixed by #23
